web-app/css/main.css

```diff
@@ -106,13 +106,13 @@ ul.one-to-many li.add {
 fieldset.embedded {
 	background-color: transparent;
 	border: 1px solid #CCCCCC;
+    box-shadow: none;
 	margin-left: 0;
 	margin-right: 0;
 	padding-left: 0;
 	padding-right: 0;
-	   -moz-box-shadow: none;
+	-moz-box-shadow: none;
```

Comment:
Looks like autoformatter is at it again.

No alphabetic sorting, certainly. Can leave as it was before, but if sorting, order by descending importance:

- Rules that influence nodes outside the element
   * display
   * position (& actual top/left values)
   * dimensions
   * box-sizing
   * margin (can go to the second group, just as dual to padding)
- Rules that influence content box
   * padding
   * border
   * alignment
   * overflow
- Content stuff
   * Everything else: font sizes/colors etc